examples/elasticsearch-operator/elasticsearch-cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: elasticsearch-cluster-sample
+spec:
+  components:
+    - type: elasticsearch-cluster
+      name: elasticsearch-cluster
+      properties:
+        version: 8.6.2
+        nodeSets:
+          - name: default
+            count: 1
+            podTemplate:
+              spec:
+                containers:
+                  - name: elasticsearch
+                    resources:
+                      requests:
+                        memory: 1Gi
+                        cpu: 1
+                      limits:
+                        memory: 1Gi
+                        cpu: 1
+            config:
+              node.store.allow_mmap: false
```

Comment:
I think we must take the volumes into consideration as it's a storage based component